x/tradeshield/migrations/v5_migrator.go

```diff
@@ -7,6 +7,7 @@ import (
 func (m Migrator) V5Migration(ctx sdk.Context) error {
 
 	m.keeper.DeleteAllPendingSpotOrder(ctx)
+	m.keeper.DeleteAllPendingPerpetualOrder(ctx)
```

Comment:
@amityadav0 you may need to create a new migrator if you want those functions calls to be called during the next testnet upgrade

Reply:
I created them in this PR https://github.com/elys-network/elys/pull/912/files, this is not yet included, is that right ?

Reply:
you are right, we are good then ;-)